### ISSUE TITLE
api.ProjectImportExport.download: The downloaded archive is invalid

### ISSUE THREAD

**Issue by alexisfrjp** [jdalrymple/gitbeaker#959]
**Description**

Using the code provided in this issue: #416, the archive tarball:
* is almost twice bigger than the one downloaded using a browser
* the archive isn't valid

```node
    api.ProjectImportExport.download(id).then(exported_project => {
      fs.promises.writeFile(output_path, exported_project).then(() => {
        printer.success('Project written to: ' + output_path)
      });
    }).catch(e => {
      console.log(e);
    })
```
Version: "@gitbeaker/node": "^17.0.3"

* Do you have any idea?

**Comment by jdalrymple** [comment]
Should be solved with #954

**Comment by alexisfrjp** [comment]
It works with `@gitbeaker/node@23.4.0`.
Thanks!
